builder/proxmox/common/step_upload_additional_isos.go

```diff
@@ -1,68 +0,0 @@
-package proxmox
-
-import (
-	"context"
-	"fmt"
-	"io"
-	"os"
-	"path/filepath"
-
-	"github.com/Telmate/proxmox-api-go/proxmox"
-	"github.com/hashicorp/packer-plugin-sdk/multistep"
-	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
-)
-
-// stepUploadAdditionalISOs uploads all additional ISO files that are mountet
-// to the VM
-type stepUploadAdditionalISOs struct{}
-
-type uploader interface {
-	Upload(node string, storage string, contentType string, filename string, file io.Reader) error
-}
-
-var _ uploader = &proxmox.Client{}
```

Comment:
This is a structural helper for making sure the step is testable. Apparently we didn't add any tests for this step (but in the original proxmox-iso builder there are tests for the "single ISO upload" step) when it was added, though.
It'd be nice to have a simple happy case test at least for the new step, or even just keeping the testability.

Reply:
Thanks for spotting that one, wasn't too sure about the testing when I initially committed.

I've now updated this PR with some test coverage which follow the "single ISO upload" step.